Makefile

```diff
@@ -106,11 +105,11 @@ install-staticcheck:
 
 .PHONY: install-goreleaser
 install-goreleaser:
-	$(call go-install-tool,$(GORELEASER),github.com/goreleaser/goreleaser@$(GORELEASER_VERSION),$(GORELEASER_VERSION))
+	$(call go-install-tool,$(GORELEASER),github.com/goreleaser/goreleaser/v2@$(GORELEASER_VERSION),$(GORELEASER_VERSION))
 
 .PHONY: install-syft
 install-syft:
-	$(call go-install-tool,$(SYFT),github.com/anchore/syft/cmd/syft@$(SYFT_VERSION),$(SYFT_VERSION))
+	curl -sSfL https://raw.githubusercontent.com/anchore/syft/main/install.sh | sh -s -- $(SYFT_VERSION)
```

Comment:
I don't think the combination `curl | sh` is acceptable. This is a very bad practice.

Reply:
? https://github.com/anchore/syft#recommended

> ## Installation
> 
> Syft binaries are provided for Linux, macOS and Windows.
> 
> ### Recommended
>
> ```bash 
> curl -sSfL https://raw.githubusercontent.com/anchore/syft/main/install.sh | sh -s -- -b /usr/local/bin
> ```

Reply:
In that case the project itself is giving extremely bad suggestions of how to install itself. `curl | sh` should **never** be done if you don't control the shell file that's downloaded yourself and are 100% sure it's ok.